configure

```diff
@@ -6938,6 +6939,21 @@ test_cpp <<EOF && enable uwp && d3d11va_extralibs="-ldxgi -ld3d11"
 #endif
 EOF
 
+# vaapi_win32 requires linking directly to dxgi if not building for
+# the desktop api partition
+test_cpp <<EOF && enable uwp && vaapi_win32_extralibs="-ldxgi"
+#ifdef WINAPI_FAMILY
+#include <winapifamily.h>
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
+#error desktop, not uwp
+#else
+// WINAPI_FAMILY_APP, WINAPI_FAMILY_PHONE_APP => UWP
```

Comment:
is that commented out on purpose? why not a C-style comments?

Reply:
Please see above at `line 6927`, a very similar check is done for the `d3d11va` backend. I think it should stay like this for consistency with those other existing checks.

Reply:
Ok. II am fine with that.